### PR TITLE
Update rnn_shakepeare.py

### PR DESCRIPTION
**Pull request by gautam1858**
Giving an existing option that is already on tensorflow so that programmers might find it useful in near future

### CODE REVIEW

File: sonnet/examples/rnn_shakespeare.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def _build(self, one_hot_input_sequence):
 
     batch_embed_module = snt.BatchApply(self._embed_module)
     input_sequence = batch_embed_module(one_hot_input_sequence)
-    input_sequence = tf.nn.relu(input_sequence)
+    input_sequence = tf.nn.relu(input_sequence, name= None)
```
Comment (esc) [inline sonnet/examples/rnn_shakespeare.py]: No spaces around the `=` when using keyword arguments.

Comment (jantee413) [inline sonnet/examples/rnn_shakespeare.py]: =

Comment (gautam1858) [inline sonnet/examples/rnn_shakespeare.py]: ok

**Comment by gautam1858** [comment]
please merge

**Pull request closed without merging**